### ISSUE TITLE
Github commit messages standard

### ISSUE THREAD

**Issue by MarcusJones** [oceanprotocol/dev-ocean#21]
What I do for summary line is to prepend a simple code from i.e. following list:
add
change
merge
fix
etc...

In my (LIMITED) experience I like to commit often, so I keep the messages to the summary line only. Further documentation I usually include in PR for feature branches... For discussion!

**Comment by jcortejoso** [comment]
I agree with @MarcusJones. A short, direct and standard subject line improve readability, mainly when the git log is presented in the oneline format. As some recommendations from https://chris.beams.io/posts/git-commit/:

> The seven rules of a great Git commit message:
> 1.     Separate subject from body with a blank line
> 2.     Limit the subject line to 50 characters
> 3.     Capitalize the subject line
> 4.     Do not end the subject line with a period
> 5.     Use the imperative mood in the subject line
> 6.     Wrap the body at 72 characters
> 7.     Use the body to explain what and why vs. how

This should be considered as a recommendation, or is preferably to force some rules using git hooks?
